app/views/groups/index.json.jbuilder

```diff
@@ -0,0 +1 @@
+json.groups @groups, :id, :name
```

Comment:
Respond to json requests asking for a list of groups (see the GroupsController#index page for more info)